Submitty/Submitty
# Initial notification settings

![new](https://user-images.githubusercontent.com/16356240/48890371-ad545600-ee06-11e8-8d1f-ce47c63da8d0.png)

The PR includes the ability for users to control their notifications.

Things to test when reviewing:
- Normal notifications still occur as expected (reply, announcement, merged threads, etc)
- Ensure notifications don't impact the users who created them
- Ensure notifications only impact users who have opted in/out
